Octokit.Tests.Integration/Reactive/Enterprise/ObservableEnterpriseOrganizationClientTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Octokit.Reactive;
+using Xunit;
+
+namespace Octokit.Tests.Integration
+{
+    public class ObservableEnterpriseOrganizationClientTests
+    {
+        readonly IObservableGitHubClient _github;
+
+        public ObservableEnterpriseOrganizationClientTests()
+        {
+            _github = new ObservableGitHubClient(EnterpriseHelper.GetAuthenticatedClient());
+        }
+
+        [GitHubEnterpriseTest]
+        public async Task CanCreateOrganization()
+        {
+            string orgLogin = Helper.MakeNameWithTimestamp("MyOrganization");
+            string orgName = String.Concat(orgLogin, " Display Name");
+
+            var newOrganization = new NewOrganization(orgLogin, EnterpriseHelper.UserName, orgName);
+            var observable = _github.Enterprise.Organization.Create(newOrganization);
```

Comment:
Of course as soon as I realised that I should read the documentation I realise that "there's no endpoint". So that's a pretty good reason to not do that. Nevermind.